apps/ledger-live-desktop/src/renderer/components/FirebaseFeatureFlags.tsx

```diff
@@ -1,4 +1,5 @@
-import React, { ReactNode } from "react";
+import React, { useState, ReactNode } from "react";
```

Comment:
Looks like `useCallback` is used but not imported

```suggestion
import React, { ReactNode, useCallback, useState } from "react";
```

Reply:
I probably messed this up today in the merge, good catch @chabroA